src/components/Proposals/ProposalDetail.js

```diff
@@ -17,12 +17,47 @@ import {
 } from '@aragon/ui'
 import { safeDiv } from '../../lib/math-utils'
 import { useGetVote } from '../../hooks/disputable-voting-logic'
+import {
+  DISPUTABLE_VOTE_STATUSES,
+  VOTE_STATUS_CANCELLED,
+  VOTE_STATUS_DISPUTED,
+  VOTE_STATUS_PAUSED,
+} from './disputable-vote-statuses'
 import DisputableActionStatus from './DisputableActionStatus'
 import InfoBoxes from './InfoBoxes'
 import SummaryBar from './SummaryBar'
+import FeedbackModule from './FeedbackModule'
 import Layout from '../Layout'
 import { networkEnvironment } from '../../current-environment'
 
+function getAttributes(status, theme) {
+  const attributes = {
+    [VOTE_STATUS_CANCELLED]: {
+      background: '#F9FAFC',
+      border: '#DDE4E9',
+      disabled: true,
+    },
+    [VOTE_STATUS_PAUSED]: {
+      background: '#fffdfa',
+      border: '#F5A623',
+      disabled: true,
```

Comment:
Do you think we could be a bit more descriptive with these property names. How about `backgroundColor`, `borderColor`, `disabledProgressBars` ?